src/Compilers/Core/Portable/WellKnownTypes.cs

```diff
@@ -600,7 +600,7 @@ internal static class WellKnownTypes
             "System.Threading.CancellationTokenSource",
 
             "System.InvalidOperationException",
-            "System.Runtime.CompilerServices.SwitchExpressionException"
+            "System.Runtime.CompilerServices.SwitchExpressionException",
```

Comment:
>", [](start = 70, length = 2)

It doesn't look like there is a meaningful change in this file. Consider reverting.